scripts/ebounty.lic

```diff
@@ -178,9 +181,9 @@
 =end
 
 require 'yaml'
+SCRIPT_NAME = script.name
```

Comment:
This should likely be `Script.current.name` to be namescoped properly with the `Script` api.

Reply:
I'll remove that. I forgot that Script.current.name would return the script name anywhere in the script